ReadASCIIApp/src/ReadASCII.cpp

```diff
@@ -93,6 +74,15 @@ ReadASCII::ReadASCII(const char *portName, const char *searchDir, const int step
     setIntegerParam(P_RampOn, 0);
     setIntegerParam(P_LookUpOn, 0);
 
+    char localDir[DIR_LENGTH], dir[DIR_LENGTH];
```

Comment:
Does `dir` need to have size `2*DIR_LENGTH + 1` ? It gets `localDir` and `/` concatenated onto it